### PR TITLE
Update DNS entry to reflect new naming scheme

### PR DESCRIPTION
**Pull request by jmorrowomni**
#  #

## 🗣 Description ##

Add CNAME entry for the Software Attestation team to point to new Landing Zone ALB.

## 💭 Motivation and context ##

Due to an External Affairs requirement, the original name can not be used.  This updates both the CNAME record and the target.

## 🧪 Testing ##

DNS resolution will be verified after the record is created.

## ✅ Pre-approval checklist ##

- [X] This PR has an informative and human-readable title.
- [X] Changes are limited to a single goal - *eschew scope creep!*
- [x] All relevant type-of-change labels have been added.
- [X] I have read the [CONTRIBUTING](../blob/develop/CONTRIBUTING.md) document.
- [X] These code changes follow [cisagov code standards](https://github.com/cisagov/development-guide).
- [x] All relevant repo and/or project documentation has been updated
      to reflect the changes in this PR.
- [x] All new and existing tests pass.

### CODE REVIEW

File: route53_swatt_app.tf [hunk 1]
```diff
@@ -0,0 +1,15 @@
+# ------------------------------------------------------------------------------
+# Resource records that support SADR
```
Comment (dav3r) [inline route53_swatt_app.tf]: I'm assuming this comment should be updated to mention whatever "swatt" is?  If it's an acronym, please include what it stands for.  Even if not, please include a brief summary of swatt.

**Review by dav3r — CHANGES_REQUESTED** [review]

File: route53_swatt_app.tf (hunk 1 above)
Comment (jmorrowomni) [reply route53_swatt_app.tf]: SWATT = SoftWare ATTestation (just an abbreviate for the project).  Will eventually remove the SADR entry as that acronym has been deemed unacceptable by EA.

Comment (jmorrowomni) [reply route53_swatt_app.tf]: @dav3r Fixed, thanks for catching that!

File: route53_swatt_app.tf [hunk 2]
```diff
@@ -0,0 +1,15 @@
+# ------------------------------------------------------------------------------
+# Resource records that support SWATT
```
Comment (jsf9k) [inline route53_swatt_app.tf]: You have to at least define what SWATT stands for.  I've never heard of it before, so I effectively don't know what this DNS record is for.

**Review by jsf9k — CHANGES_REQUESTED** [review]

**Review by dav3r — APPROVED** [review]
👍

File: route53_swatt_app.tf (hunk 1 above)
Comment (jmorrowomni) [reply route53_swatt_app.tf]: See [18c9f13](https://github.com/cisagov/cool-dns-cyber.dhs.gov/pull/88/commits/18c9f13ba2fe6c83b99b5dbcb26ddd8dcc323cab)

File: route53_swatt_app.tf (hunk 2 above)
Comment (jmorrowomni) [reply route53_swatt_app.tf]: See [18c9f13](https://github.com/cisagov/cool-dns-cyber.dhs.gov/pull/88/commits/18c9f13ba2fe6c83b99b5dbcb26ddd8dcc323cab)

**Review by jsf9k — APPROVED** [review]

**Comment by dav3r** [comment]
To test this, I applied it and it looks good to me.  @jmorrowomni can you please confirm?  Once you do, I will merge this PR.

**Comment by jmorrowomni** [comment]
> To test this, I applied it and it looks good to me. @jmorrowomni can you please confirm? Once you do, I will merge this PR.

Looks good!

image

**Pull request merged**